fam_change_permissions.html

```diff
@@ -161,7 +169,7 @@ <h2>Example</h2>
 		</article>
 	</section>
 	<footer>
-		<p>Copyright 2019, Hewlett Packard Enterprise Development Co, LLP</p>
```

Comment:
Line 132, Notes should say that permissions are checked on first descriptor use in a running program. Changed permissions are only applied the next time the program is run, not immediately on next access.